### PR TITLE
[ID] Add `osu!_wiki`

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_wiki/id.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- [Panduan kontribusi](/wiki/osu!_wiki/Contribution_guide)
+- [Pemeliharaan](/wiki/osu!_wiki/Maintenance)
+
+## Sejarah
+
+*Halaman utama: [Sejarah osu! wiki](/wiki/History_of_osu!/osu!_wiki)*
+
+[Upaya pertama](https://osu.ppy.sh/community/forums/posts/1175876) untuk membuat basis pengetahuan terstruktur dimulai pada tahun 2011, saat [peppy](/wiki/People/peppy) membuat instansi [MediaWiki](https://en.wikipedia.org/wiki/MediaWiki) bagi siapa saja yang mau berpartisipasi. Selama bertahun-tahun, pemeliharaan dan pengoperasian MediaWiki telah terbukti sulit dan melelahkan, dan pada [26 Agustus 2016](https://discord.com/channels/188630481301012481/218677502141399041/218678017659109376) peppy dan tim sukarelawan memulai proses migrasi osu! wiki ke GitHub, di mana osu! wiki berada sampai hari ini. Wiki sejak itu telah diintegrasikan ke dalam situs web osu! yang baru dan menerima pembaruan teknis dan visual yang penting untuk pengalaman pengguna yang baik.
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:14]:
```suggestion
[Upaya pertama](https://osu.ppy.sh/community/forums/posts/1175876) untuk membuat basis pengetahuan terstruktur dimulai pada tahun 2011, saat [peppy](/wiki/People/peppy) membuat instansi [MediaWiki](https://en.wikipedia.org/wiki/MediaWiki) bagi siapa saja yang mau berpartisipasi. Selama bertahun-tahun, pemeliharaan dan pengoperasian MediaWiki telah terbukti sulit dan melelahkan, dan pada [26 Agustus 2016](https://discord.com/channels/188630481301012481/218677502141399041/218678017659109376) peppy dan tim sukarelawan memulai proses migrasi osu! wiki ke GitHub, di mana osu! wiki berada sampai hari ini. Sejak itu, wiki telah diintegrasikan ke dalam situs web osu! yang baru dan menerima pembaruan teknis dan visual yang penting untuk pengalaman pengguna yang baik.
```

File: wiki/osu!_wiki/id.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Struktur
+
+Pintu gerbang ke osu! wiki adalah [halaman utama](/wiki/Main_Page), yang dibagi menjadi beberapa kategori tingkat tinggi, masing-masing memiliki deskripsi umum dan mengandung daftar artikel penting. Artikel yang tidak terdaftar di halaman utama, yang biasanya lebih spesifik, dapat dijangkau baik dari halaman lain menggunakan inline link, atau melalui fitur pencarian situs web.
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:18]:
```suggestion
Pintu gerbang ke osu! wiki adalah [halaman utama](/wiki/Main_Page), yang dibagi menjadi beberapa kategori tingkat tinggi, masing-masing memiliki deskripsi umum dan mengandung daftar artikel penting. Artikel yang tidak terdaftar di halaman utama, yang biasanya lebih spesifik, dapat dijangkau baik dari halaman lain menggunakan tautan dalam artikel atau melalui fitur pencarian situs web.
```

File: wiki/osu!_wiki/id.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Sebuah artikel menjelaskan satu istilah atau konsep dan memberikan gambaran umum tentang topik lain yang relevan. Agar lebih baik dalam menggambarkan sebuah subjek, artikel dibagi menjadi subbagian yang mencakup bagian yang berbeda dari subjek tersebut. Artikel juga dapat mencakup tangkapan layar yang relevan atau penjelasan visual lainnya, atau tautan ke artikel lain tentang topik tersebut.
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:20]:
```suggestion
Sebuah artikel menjelaskan satu istilah atau konsep dan memberikan gambaran umum tentang topik lain yang relevan. Agar lebih baik dalam menggambarkan sebuah subjek, artikel dibagi menjadi sub-bagian yang mencakup bagian yang berbeda dari subjek tersebut. Artikel juga dapat mencakup tangkapan layar yang relevan atau penjelasan visual lainnya, atau tautan ke artikel lain tentang topik tersebut.
```

File: wiki/osu!_wiki/id.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Terjemahan dalam bahasa yang berbeda tersedia untuk beberapa artikel. Terjemahah tersebut dibuat oleh anggota komunitas, dan pilihan bahasa hanya dibatasi oleh [daftar bahasa](/wiki/Article_styling_criteria/Formatting#locales) yang didukung oleh situs web itu sendiri.
+
+## Model
+
+Osu! wiki dibangun berdasarkan konsep aksesibilitas: pengeditan dan terjemahan dianjurkan dan dapat dilakukan oleh siapa saja yang mampu dan berpengetahuan. Data untuk artikel disimpan di [repositori GitHub](https://github.com/ppy/osu-wiki) dan diperbarui oleh kontributor melalui [*pull request*](/wiki/osu!_wiki/Contribution_guide), yang kemudian ditinjau dan di-*merge* oleh [pengelola wiki](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:26]:
```suggestion
osu! wiki dibangun berdasarkan konsep aksesibilitas: pengeditan dan terjemahan dianjurkan dan dapat dilakukan oleh siapa saja yang mampu dan berpengetahuan. Data untuk artikel disimpan di [repositori GitHub](https://github.com/ppy/osu-wiki) dan diperbarui oleh kontributor melalui [*pull request*](/wiki/osu!_wiki/Contribution_guide), yang kemudian ditinjau dan di-*merge* oleh [pengelola wiki](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
```

File: wiki/osu!_wiki/id.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Artikel diformat menggunakan [*GitHub-flavoured Markdown*](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Selain itu, mesin situs web mengimplementasikan fitur khusus yang penting untuk wiki secara keseluruhan, seperti pengalihan, halaman pertama, atau perbaikan pemformatan.
+
+Artikel disusun menurut [kriteria gaya artikel](/wiki/Article_styling_criteria) dari osu! wiki, yang berarti artikel ditulis dalam bahasa netral dan mengikuti aturan format dan desain umum. Beberapa artikel informasi berfungsi sebagai daftar pertanyaan yang sering diajukan (frequently asked question (FAQ)) tentang topik tersebut dan menggunakan bahasa informal. Terjemahan mengikuti prinsip persamaan konten dan mematuhi aturan pemformatan yang sama.
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:30]:
```suggestion
Artikel disusun menurut [kriteria gaya artikel](/wiki/Article_styling_criteria) dari osu! wiki, yang berarti artikel ditulis dalam bahasa netral dan mengikuti aturan format dan desain umum. Beberapa artikel informasi berfungsi sebagai daftar pertanyaan yang sering diajukan (*frequently asked question* (FAQ)) tentang topik tersebut dan menggunakan bahasa informal. Terjemahan mengikuti prinsip persamaan konten dan mematuhi aturan pemformatan yang sama.
```

File: wiki/osu!_wiki/id.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+Meskipun memiliki akhiran *wiki* dalam namanya, osu! wiki tidak sepenuhnya memenuhi [konsep Wiki](https://id.wikipedia.org/wiki/Wiki):
+
+- Pengguna didorong untuk meningkatkan artikel yang ada dan membuat yang baru, tetapi pengguna diharapkan memiliki pengetahuan dasar tentang [GitHub](https://github.com), yang meningkatkan ambang batas yang diperlukan untuk masuk.
+- Semua perubahan pada osu! wiki telah dimoderasi sebelumnya dan menjalani proses peninjauan, di mana seorang kontributor diharapkan untuk berkolaborasi dengan pengelola wiki dan menanggapi umpan balik.
+
+## Lisensi
+
+Sebagian besar konten di osu! wiki dilisensikan di bawah [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Ini berarti bahwa secara umum, materi wiki dapat digunakan secara non-komersial (lihat [ringkasan](https://tldrlegal.com/license/creative-commons-attribution-noncommercial-4.0-international-(cc-by-nc-4.0))). Beberapa konten mungkin dicakup oleh lisensi pihak ketiga yang berbeda, yang kemudian akan dikutip secara lokal dan menggantikan lisensi global.
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:39]:
```suggestion
Sebagian besar konten di osu! wiki dilisensikan di bawah [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Ini berarti bahwa secara umum, materi wiki dapat digunakan secara non-komersial (lihat [ringkasan lisensi](https://tldrlegal.com/license/creative-commons-attribution-noncommercial-4.0-international-(cc-by-nc-4.0))). Beberapa konten mungkin dicakup oleh lisensi pihak ketiga yang berbeda, yang kemudian akan dikutip secara lokal dan menggantikan lisensi global.
```

File: wiki/osu!_wiki/id.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+Harap dicatat bahwa ini tidak mencakup penggunaan "osu!" atau merek "ppy" dalam perangkat lunak, sumber daya, iklan, atau promosi apa pun, karena hal ini dilindungi oleh undang-undang merek dagang. Jika Anda memerlukan izin untuk penggunaan persyaratan ini, silakan [hubungi kami](mailto:contact@ppy.sh).
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/id.md:41]:
```suggestion
Harap dicatat bahwa ini tidak mencakup penggunaan "osu!" atau merek "ppy" dalam perangkat lunak, sumber daya, iklan, atau promosi apa pun, karena hal ini dilindungi oleh undang-undang merek dagang. Jika kamu memerlukan izin untuk penggunaan persyaratan ini, silakan [hubungi kami](mailto:contact@ppy.sh).
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**